src/Uno.UI.RuntimeTests/MUX/Utilities/TestHelpers.cs

```diff
@@ -139,7 +139,7 @@ namespace MUXControlsTestApp.Utilities
 {
 	public static class TestUtilities
 	{
-		public static int DefaultWaitMs = Debugger.IsAttached ? 120000 : 5000;
```

Comment:
The point of this particular line is to make it possible to break inside a test for a little while and still make it pass. Is it failing otherwise?

Reply:
I understand, but I'm still very against any `Debugger.IsAttached` checks. We can put some commented-out options that can be turned on on demand, but leaving these checks by default has burnt me way too many times (not this specific instance, but it's the same point).